server/actions/list_secrets.go

```diff
@@ -1,11 +1,40 @@
 package actions
 
 import (
+	"os"
+	"strings"
+
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/secretsmanager"
 )
 
+var (
+	nameFilters *secretsmanager.Filter
+)
+
+func init() {
```

Comment:
I think we shouldn't use `init()` in package actions. Try a function `getFilterNames()` and use it with a singleton to call from GetListSecrets() instead